controllers/grid/users/stageParticipant/form/AddParticipantForm.inc.php

```diff
@@ -82,6 +82,28 @@ function fetch($request) {
 		// assign the vars required for the request
 		$templateMgr->assign('submissionId', $this->getSubmission()->getId());
 
+		// If submission is in review, add a list of reviewer Ids that should not be
+		// assigned as participants because they have blind peer reviews in progress
+		import('lib.pkp.classes.submission.reviewAssignment.ReviewAssignment');
+		$blindReviewerIds = array();
+		if ($this->getSubmission()->getStageId() <= WORKFLOW_STAGE_ID_EXTERNAL_REVIEW) {
```

Comment:
should here actually be only internal and external review? it would consider also submisison stage, or?

Reply:
Agreed, and I'd suggest avoiding numerical comparisons on stage IDs. Better to use `in_array($stageId, array(WORKFLOW_STAGE_ID_INTERNAL_REVIEW, WORKFLOW_STAGE_ID_EXTERNAL_REVIEW))` IMO.

Reply:
Good idea. I was thinking of how you can move forwards and backwards in places in the workflow (for example, you can forward to copyediting, then go back to submission stage and forward to review, to go "back"). But it occurs to me that it isn't currently possible to go back to the submission stage in this way.

I think an easier way to go back has also been a requested feature in the past. But I can see all kinds of complications arising here from a practice we discourage, so it's best to just to leave it for the review stages.